src/terrain/clients/iplant_groups.clj

```diff
@@ -63,7 +63,8 @@
 
 (defn- create-folder [client user name]
   (c/add-folder client (config/grouper-user) name "")
-  (c/grant-folder-privilege client (config/grouper-user) name user :stem)
+  (when-not (= user (config/grouper-user))
+    (c/grant-folder-privilege client (config/grouper-user) name user :stem))
```

Comment:
This change simply skips the permission grant if the user already has permission on the new folder.